app/src/main/assets/OpenAPSSMBDynamicISF/determine-basal.js

```diff
@@ -501,9 +510,41 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         return rT;
     }
 
-    // min_bg of 90 -> threshold of 65, 100 -> 70 110 -> 75, and 130 -> 85
-    var threshold = min_bg - 0.5*(min_bg-40);
+    // min_bg of 90 -> threshold of 65, 100 -> 70 110 -> 75, and 130 -> 85, or if specified by user, take that value
+        var lgsThreshold = profile.lgsThreshold;
+        console.error("Profile LGS Threshold set to"+lgsThreshold+"; ");
+        //if(lgsThreshold < 5.6)
+        if(lgsThreshold > 3.2 && lgsThreshold < 5.6){
```

Comment:
All JS code code works in mgdl. I believe we must continue with this assumption and do conversion in kotlin

Reply:
I've tried that. The conversion doesn't pick up using the Kotlin code. It only changes the display value and doesn't get passed to determine basal. 

Happy to have it in Kotlin, and would prefer it that way, but so far haven't been able to make it work.